### PR TITLE
Set command center when playing

### PR DESCRIPTION
**Pull request by raphrel**

### CODE REVIEW

File: Sources/Core/Components/CommandCenter.swift [hunk 1]
```diff
@@ -27,50 +27,70 @@
...
         remote.previousTrackCommand.isEnabled = false
         remote.nextTrackCommand.isEnabled = false
-
+        
+        if #available(iOS 9.1, *) {
+            remote.changePlaybackPositionCommand.isEnabled = true
+        }
+    }
+    
+    private func playCommand(player: MediaPlayer) {
```
Comment (solevic) [inline Sources/Core/Components/CommandCenter.swift]: naming could be more explicit

File: Sources/Core/Components/CommandCenter.swift [hunk 2]
```diff
@@ -27,50 +27,70 @@
...
+public struct ModernAVPlayerCommandCenter: CommandCenter {
+    public let remote: MPRemoteCommandCenter
+    
+    public init(remote: MPRemoteCommandCenter = MPRemoteCommandCenter.shared()) {
         self.remote = remote
-        defaultSetup()
     }
     
-    private func defaultSetup() {
+    public func configure(player: MediaPlayer) {
```
Comment (solevic) [inline Sources/Core/Components/CommandCenter.swift]: doc

File: Sources/Core/Components/CommandCenter.swift [hunk 3]
```diff
@@ -27,50 +27,70 @@
...
-    public init(player: ModernAVPlayer, remote: MPRemoteCommandCenter = MPRemoteCommandCenter.shared()) {
-        self.player = player
+    init(remote: MPRemoteCommandCenter)
+    func configure(player: MediaPlayer)
+}
+
+public struct ModernAVPlayerCommandCenter: CommandCenter {
+    public let remote: MPRemoteCommandCenter
+    
+    public init(remote: MPRemoteCommandCenter = MPRemoteCommandCenter.shared()) {
```
Comment (solevic) [inline Sources/Core/Components/CommandCenter.swift]: doc

File: Sources/Core/ModernAVPlayer.swift [hunk 4]
```diff
@@ -50,18 +50,22 @@ public final class ModernAVPlayer: NSObject, MediaPlayer {
...
-    // MARK: - Variable
+    // MARK: - Variables
     
     private let context: ModernAVPlayerContext
+    private let commandCenter: CommandCenter
     
     // MARK: - Init
     
-    public init(config: PlayerConfiguration = ModernAVPlayerConfiguration(), plugins: [PlayerPlugin] = []) {
+    public init(config: PlayerConfiguration = ModernAVPlayerConfiguration(),
```
Comment (solevic) [inline Sources/Core/ModernAVPlayer.swift]: doc

File: Sources/Core/Components/CommandCenter.swift [hunk 5]
```diff
@@ -27,50 +27,70 @@
...
+    private func playCommand(player: MediaPlayer) {
         remote.playCommand.addTarget { [player] _ -> MPRemoteCommandHandlerStatus in
             LoggerInHouse.instance.log(message: "Remote command: play", event: .info)
             player.play()
             return .success
         }
-        
+    }
+    
+    private func pauseCommand(player: MediaPlayer) {
```
Comment (RealBug) [inline Sources/Core/Components/CommandCenter.swift]: Remame function addPauseCommand ?

File: Sources/Core/ModernAVPlayer.swift [hunk 6]
```diff
@@ -94,11 +98,15 @@ public final class ModernAVPlayer: NSObject, MediaPlayer {
     - parameter shouldPlaying: play after media is loaded
     */
     public func loadMedia(media: PlayerMedia, shouldPlaying: Bool) {
+        if shouldPlaying {
```
[the remark this answers is not in the document]
Comment (raphrel) [reply Sources/Core/ModernAVPlayer.swift]: ok

File: Sources/Core/Components/CommandCenter.swift (hunk 1 above)
Comment (raphrel) [reply Sources/Core/Components/CommandCenter.swift]: set to -> setPlayCommandTarget()

File: Sources/Core/ModernAVPlayer.swift (hunk 4 above)
Comment (raphrel) [reply Sources/Core/ModernAVPlayer.swift]: done

File: Sources/Core/Components/CommandCenter.swift (hunk 3 above)
Comment (raphrel) [reply Sources/Core/Components/CommandCenter.swift]: done

File: Sources/Core/Components/CommandCenter.swift (hunk 2 above)
Comment (raphrel) [reply Sources/Core/Components/CommandCenter.swift]: done

File: Sources/Core/Components/CommandCenter.swift (hunk 5 above)
Comment (raphrel) [reply Sources/Core/Components/CommandCenter.swift]: I used setPauseCommandCallback

**Pull request merged**